docs/content/preview/develop/change-data-capture/using-logical-replication/_index.md

```diff
@@ -118,3 +118,8 @@ For reference documentation, see [YugabyteDB Connector](./yugabytedb-connector/)
 - Support for tablet splitting with logical replication is disabled from v2024.1.4 and v2024.2.1. Tracked in issue [24918](https://github.com/yugabyte/yugabyte-db/issues/24918).
 
 - A replication slot should be consumed by at most one consumer at a time. However, there is currently no locking mechanism to enforce this. As a result, you should ensure that multiple consumers do not consume from a slot simultaneously. Tracked in issue [20755](https://github.com/yugabyte/yugabyte-db/issues/20755).
+
+- Limitation on updating or deleting a row inserted in the same transaction:
```

Comment:
suggest:
- If a row is updated or deleted in the same transaction in which it was inserted, CDC cannot retrieve the before-image values for the UPDATE / DELETE event. If the replica identity is not CHANGE, then CDC will throw an error while processing such events.

    To handle updates/deletes with a non-CHANGE replica identity, set the YB-TServer flag cdc_send_null_before_image_if_not_exists to true. With this flag enabled, CDC will send a null before-image instead of failing with an error.